src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -28,7 +28,12 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import java.util.logging.Logger;
+<<<<<<< HEAD
+import com.google.sps.utils.User;
+import com.google.appengine.api.datastore.Query.SortDirection;
+=======
 import com.google.sps.utils.QuizTimingPropertiesUtils;
+>>>>>>> a4f50e37510867b1aba9cb0c5a74f93d45725139
```

Comment:
This is a merge conflict you need to clean up

Reply:
got it, sorry about that